helm/alfresco-content-services/templates/config-transformer-routes.yaml

```diff
@@ -4,8 +4,6 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ template "alfresco.shortname" . }}-transform-routes-configmap
-  labels:
-    {{- include "transform-router.labels" . | nindent 4 }}
```

Comment:
why removing this without a replacement? we will forget for sure once we migrate it otherwise

Reply:
These labels are not used as far as I can tell so I decided to remove them